src/bors/handlers/workflow.rs

```diff
@@ -77,6 +78,21 @@ pub(super) async fn handle_workflow_completed(
         branch: payload.branch,
         commit_sha: payload.commit_sha,
     };
+
+    if let Some(running_time) = payload.running_time {
```

Comment:
When we find out that the workflow ran too quickly, we should change its status to failed, and then act as it was an actual failure (with some custom error code).